packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -460,10 +460,14 @@ class InspectorService extends InspectorServiceBase {
     final response = await invokeServiceMethodDaemonNoGroupArgs(
       'getPubRootDirectories',
     );
-
-    return (response as List<dynamic>)
-        .map<String>((e) => e.toString())
-        .toList();
+    if (response.runtimeType == List) {
```

Comment:
or just `if (response == nul) return [];`

Reply:
I believe the response is `{result: null}` so I don't think this would work.